homeassistant/components/energy/websocket_api.py

```diff
@@ -64,13 +63,15 @@ async def async_get_energy_platforms(
 
     @callback
     def _process_energy_platform(
-        hass: HomeAssistant, domain: str, platform: ModuleType
+        hass: HomeAssistant,
+        domain: str,
+        platform: EnergyPlatform,
     ) -> None:
         """Process energy platforms."""
         if not hasattr(platform, "async_get_solar_forecast"):
             return
```

Comment:
Not sure if this is needed anymore. The function was present in the only `energy` I could find in core. Maybe I'm missing some.?

https://github.com/home-assistant/core/blob/1559562c267c52268004b4ed90e51f891e0b8289/homeassistant/components/forecast_solar/energy.py#L10-L12

Reply:
I guess someone could name their file `energy.py` and we might try to load it?